ballot/urls/ballot.py

```diff
@@ -1,11 +1,12 @@
-from django.conf.urls import patterns, url
+from rest_framework.routers import SimpleRouter
+# from django.conf.urls import patterns, url
 from django.contrib import admin
 
 from ..views import ballot as views
 
 admin.autodiscover()
 
-urlpatterns = patterns(
-    '',
-    url(r'^locality/(?P<locality_id>[0-9]+)/ballot/$', views.ballot_view,
-        name='locality_ballot'))
+api = SimpleRouter()
+api.register(r'locality/(?P<locality_id>[0-9]+)', views.BallotViewSet,
+             base_name='ballot')
```

Comment:
I think this broke the tests? Anyway, something did -- the failure looks related.